docs/src/main/asciidoc/grpc-service-consumption.adoc

```diff
@@ -204,7 +204,7 @@ public class StreamingEndpoint {
 
 For each gRPC service you inject in your application, you can configure the following attributes:
 
-include::{generated-dir}/config/quarkus-grpc_quarkus.grpc-client.adoc[opts=optional, leveloffset=+1]
+include::{generated-dir}/config/quarkus-grpc_quarkus.grpc.clients.adoc[opts=optional, leveloffset=+1]
```

Comment:
This looks odd.

Reply:
I believe is what it was supposed to be. The [original](https://quarkus.io/guides/grpc-service-consumption#client-configuration) only displays the `quarkus.grpc-client` options, which is incorrect, based on the paragraph below (it mentions `client-name` which only exists in quarkus-grpc_quarkus.grpc-client.adoc)

Reply:
Yeah, I think it's just the original config being wrong with one config property under `quarkus.grpc-client` and all the others under `quarkus.grpc.clients`.

Not sure how to handle that. You could keep the `quarkus.grpc-client` root and mark the property as deprecated (and use an Optional) and move the property to the other root.

I think it warrants a discussion with the original authors to check if it was intended or just a mistake, though.

Reply:
I meant that the property is valid, the `.adoc` chosen is what's wrong here IMHO.

Reply:
@alesj or @cescoffier can clarify, but AFAIK it's meant to be a global property for clients (which are configured individually under `quarkus.grpc.clients`)

Reply:
OK so don't we need both config files to be included in the doc then? If they are both related to clients?

Reply:
Maybe, yes. I'll add both